telemetry/glean/metrics.yaml

```diff
@@ -229,82 +275,115 @@ ad_client:
       - https://bugzilla.mozilla.org/show_bug.cgi?id=1900898
     expires: never
 
-interaction:
-  kevel:
-    type: event
+  served_timestamp:
+    type: datetime
     description: >
-      Captures specific Kevel fields.
+      Timestamp indicating when the ad was served by the Unified API to client.
+      This timestamp along with fetched_timestamp indicates the freshness of
+      the ad.  If the ad is not cached on the server these two timestamps will
+      differ by milliseconds. Should not be null.
+    lifetime: application
+    send_in_pings:
+      - amp-interaction
+      - kevel-interaction
+      - request-stats
```

Comment:
This one I wasn't sure if we'd care about -- glean should have an ingestion timestamp as well, so by using this field I think we'd have slightly more consistency since the ingestion would occur a bit after we handle the request

Reply:
I've confirmed that the `submission_timestamp` will not match the `served_timestamp`.  By using `served_timestamp` across both pings we will ensure that any time series analysis of serving and impression/clicks will be cleaner.